account_easy_reconcile/easy_reconcile.py

```diff
@@ -0,0 +1,424 @@
+    def _get_rec_method(self):
+        return self._get_all_rec_method()
+
+    name = fields.Selection('_get_rec_method', string='Type', required=True)
+    sequence = fields.Integer(string='Sequence',
+                              default=1,
+                              required=True,
+                              help="The sequence field is used to order "
+                              "the reconcile method"
+                              )
+    task_id = fields.Many2one('account.easy.reconcile',
+                              string='Task',
+                              required=True,
+                              ondelete='cascade'
+                              )
+    company_id = fields.Many2one('res.company',
+                                 string='Company',
+                                 related="task_id.company_id",
+                                 store=True,
+                                 readonly=True
+                                 )
+
+#     def init(self, cr):
+#         """ Migration stuff
+#
+#         Name is not anymore methods names but the name
+#         of the model which does the reconciliation
+#         """
+#         cr.execute("""
+#         UPDATE account_easy_reconcile_method
+#         SET name = 'easy.reconcile.simple.partner'
+#         WHERE name = 'action_rec_auto_partner'
+#         """)
+#         cr.execute("""
+#         UPDATE account_easy_reconcile_method
+#         SET name = 'easy.reconcile.simple.name'
+#         WHERE name = 'action_rec_auto_name'
+#         """)
+
+
+class AccountEasyReconcile(models.Model):
+
+    _name = 'account.easy.reconcile'
+    _inherit = ['mail.thread']
+    _description = 'account easy reconcile'
+
+    @api.one
+    @api.depends('account')
+    def _get_total_unrec(self):
+        obj_move_line = self.env['account.move.line']
+        self.unreconciled_count = len(obj_move_line.search(
+            [('account_id', '=', self.account.id),
+             ('reconcile_id', '=', False),
+             ('reconcile_partial_id', '=', False)],
+            ))
+
+    @api.one
+    @api.depends('account')
+    def _get_partial_rec(self):
+        obj_move_line = self.env['account.move.line']
+        self.reconciled_partial_count = len(obj_move_line.search(
+            [('account_id', '=', self.account.id),
+             ('reconcile_id', '=', False),
+             ('reconcile_partial_id', '!=', False)],
+            ))
+
+    @api.one
+    @api.depends('history_ids')
+    def _last_history(self):
+        # do a search() for retrieving the latest history line,
+        # as a read() will badly split the list of ids with 'date desc'
+        # and return the wrong result.
+        history_obj = self.env['easy.reconcile.history']
+        last_history_rs = history_obj.search(
+            [('easy_reconcile_id', '=', self.id)],
+            limit=1, order='date desc'
+            )
+        self.last_history = last_history_rs[0] if last_history_rs else False
```

Comment:
`last_history_rs == last_history_rs[0]` if not empty

so `last_history_rs or False` would be enought